tests/integration/test_cladetime_integration.py

```diff
@@ -1,60 +1,28 @@
-import lzma
 from datetime import datetime, timezone
 from unittest.mock import MagicMock, patch
 
 import polars as pl
 import pytest
-import requests
 from freezegun import freeze_time
 from polars.testing import assert_frame_equal, assert_frame_not_equal
 
 from cladetime import CladeTime, sequence
 from cladetime.exceptions import CladeTimeSequenceWarning
-from cladetime.util.config import Config
-from cladetime.util.reference import _docker_installed, _get_s3_object_url
+from cladetime.util.reference import _docker_installed
 
 docker_enabled = _docker_installed()
 
 
-@pytest.fixture()
-def metadata_100k(tmp_path) -> pl.LazyFrame:
-    "Return metadata for Nextstain's 100k samples as of 2024-11-01"
-    config = Config()
-    metadata_url = _get_s3_object_url(
-        bucket_name=config.nextstrain_ncov_bucket,
-        object_key="files/ncov/open/100k/metadata.tsv.xz",
-        date=datetime(2024, 11, 1, tzinfo=timezone.utc),
-    )[1]
-
-    # download test metadata for Nextstrain's 100k samples (we can't use polars to scan it from
-    # s3 like we usually do, because the 100k samples don't have ZSTD-compressed versions
-    # and lmza.open doesn't support https)
```

Comment:
"lzma" typo?

Reply:
Thanks! Will fix this up in the next PR!